miden-crypto/src/merkle/smt/full/concurrent/mod.rs

```diff
@@ -710,6 +710,9 @@ fn fetch_sibling_pair(
 }
 
 #[cfg(feature = "internal")]
+/// Builds a subtree from the given leaves for benchmarking purposes.
+///
+/// Returns a tuple containing the constructed inner nodes and the final root leaf.
```

Comment:
here and in other places too, we usually put attributes after comments, could we update this?